migrate2rocky/migrate2rocky.sh

```diff
@@ -188,8 +188,11 @@ exit_clean () {
 }
 
 pre_check () {
-    if [[ -e /etc/rhsm/ca/katello-server-ca.pem ]]; then
-	exit_message "Migration from Katello-modified systems is not supported by migrate2rocky."
+    if [[ -e /etc/rhsm/ca/katello-server-ca.pem ]] || [[ -s /etc/yum.repos.d/redhat.repo ]]; then
+	    exit_message "Migration from Katello-modified systems is not supported by migrate2rocky. See the README file for details."
```

Comment:
You're still exiting on redhat.repo.  This will break the migration from RHEL 8.

Reply:
But isn't this now only exiting, as long as Katello put its certificate in the RHSM folder? Which means it only exits if it is configured with Katello and has any applied repos from subscription-manager, right?

Reply:
Check your logic, you know what boolean || means, right?

Reply:
Current version of this PR looks good to me.